mars/oscar/backends/ray/pool.py

```diff
@@ -107,7 +120,8 @@ async def _kill_actor_forcibly(self, process: 'ray.actor.ActorHandle'):
 
     async def is_sub_pool_alive(self, process: 'ray.actor.ActorHandle'):
         try:
-            await process.health_check.remote()
+            # Check for liveness, any property will be ok.
+            await process.actor_pool.remote('process_index')
```

Comment:
It's a bit hack, can we add more comments here to explain why calling the `actor_pool` with the `process_index`?